src/Windows/Avalonia.Win32/Win32StorageProvider.cs

```diff
@@ -236,7 +232,7 @@ private static byte[] FiltersToPointer(IReadOnlyList<FilePickerFileType>? filter
                 {
                     var filterStr = new UnmanagedMethods.COMDLG_FILTERSPEC
                     {
-                        pszName = filter.Name ?? string.Empty,
```

Comment:
IIRC this was added because of native crashes caused by invalid user code. We can somewhat trust annotations in the managed world since it would just cause an NRE, but not when passing things to the native code.

Reply:
The filter can't be null here: it's enforced in `FilePickerFileType` ctor instead so all usages are safe.